src/SFA.DAS.ApplyService.InternalApi/Services/AssessorLookupService.cs

```diff
@@ -1,4 +1,6 @@
 ﻿using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.Domain.Sectors;
+using static SFA.DAS.ApplyService.Application.Apply.Roatp.RoatpWorkflowPageIds.DeliveringApprenticeshipTraining;
```

Comment:
Using this static has made a lot of the constants in the case statements look really weird. Do you really need it?

Reply:
No, but it makes my stuff REALLY long.  However, I may rejig the class structures to be shorter